src/middlewared/middlewared/plugins/smb.py

```diff
@@ -1253,7 +1257,7 @@ async def share_precheck(self, data):
                 )
 
         if data.get(share_field.NAME) is not None:
-            await self.validate_share_name(data[share_field.NAME], 'sharing.smb.share_precheck', verrors, False)
+            await self.validate_share_name(data[share_field.NAME], 'sharing.smb.share_precheck', verrors)
```

Comment:
FWIW, I don't see any docs as to what `share_precheck` is, so I don't know whether leaving `old_id` default to `None` here is correct.

Reply:
I added a comment. It's a method that Evgeny requested for UI purposes.